contracts/Core/StakeManager.sol

```diff
@@ -198,19 +197,16 @@ contract StakeManager is Initializable, ACL, StakeStorage, StateManager, Pause,
         require(lock.withdrawAfter + parameters.withdrawReleasePeriod() >= epoch, "Release Period Passed"); // Can Use ResetLock
 
         uint256 withdrawAmount = lock.amount;
-
+        uint256 commission = lock.commission;
         if (lock.commission > 0) {
-            withdrawAmount = lock.amount - lock.commission;
```

Comment:
if statement not required
commission variable not required

Reply:
Yes. I will remove it

Reply:
commission variable is needed since slither expect us to do transfer after reset lock as happened. So we need variable to store lock.comission. But If condition can be removed.

Reply:
hmm we had added this if, in case if resetLock is being called by Staker, where lock.commission is zero
if we remove if, redundant transfer call is made,
its just a optimisation.

Now its our call if we should have it or not.

CC : @hrishikeshio @SkandaBhat @SamAg19 @adi44

Reply:
hmm but one can say there are 50 delegators and 1 staker 
so adding cost of if for all 50 delegators 
vs adding cost for 1 staker